### DISCUSSION TITLE
Save game folder

### DISCUSSION THREAD

**Discussion by Hipshot-zz** [Keriew/augustus#879] [Ideas]
I was looking around a bit and could not find a way to set the save game folder, I did look in the augustus.ini, but maybe I missed it?

It would be a nice addition otherwise, I can just set a folder in like dropbox or whatever and then have that as a cheap mans cloud save. I do this with a lot of games, often using symlinks, but I rather not sync the entire project, only a folder or so.

**Comment by crudelios** [comment]
This has been planned for ages but I never got around to doing it.

**Comment by Hipshot-zz** [comment]
Good to hear. Maybe in the future then!

**Comment by all-zer** [comment]
The folders is not so importants... Better to have same file extensions... *.SAV
To be able to open in different versions.

For example,,, till yesterday, i liked this augustus version. But i noticed, that there is no gardens... and a lot of buildings, that i even not understaind, how to use it, and what effect it does?
There is no info on wiki-pages...
